cmd/nerdctl/image/image_inspect_test.go

```diff
@@ -94,16 +90,16 @@ func TestImageInspectDifferentValidReferencesForTheSameImage(t *testing.T) {
 			nerdtest.Private,
 		),
 		Setup: func(data test.Data, helpers test.Helpers) {
-			helpers.Ensure("pull", "alpine", "--platform", platform)
-			helpers.Ensure("pull", "busybox", "--platform", platform)
-			helpers.Ensure("pull", "busybox:stable", "--platform", platform)
-			helpers.Ensure("pull", "registry-1.docker.io/library/busybox", "--platform", platform)
-			helpers.Ensure("pull", "registry-1.docker.io/library/busybox:stable", "--platform", platform)
```

Comment:
Can we just remove this line and then keep using the official image?

Reply:
Sure, we can.
That is not just this line though - stable is used throughout.